sails/config/policies.js

```diff
@@ -9,6 +9,9 @@
  */
 
 module.exports.policies = {
-
     //'*': 'is-logged-in',
```

Comment:
Assumption: isLoggedIn policy is deactived for development environment
-> It should be moved to env/production?